.gitignore

```diff
@@ -5,7 +5,7 @@ vendor
 .idea/
 coverage.clover
 tests/coverage
-.php_cs.cache
+.php-cs-fixer.cache
```

Comment:
This was still using the old named after it was renamed as part of 1a1e0bd3d29f2b91fd7c436bc79cec8fb9941a97.